<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateClients</class>
 <widget class="QDialog" name="UpdateClients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование информации о клиенте</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPushButton" name="backToViewInfoButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #dbdcf7;
	border: 1px solid #999999;
}

QPushButton:hover {
	background-color: #d5d6f0;
	border: 1px solid #acacac;
}

QPushButton:pressed {
	background-color: #cbcde6;
	border: 2px solid #999999;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/backToView.png</normaloff>:/images/backToView.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="contactsLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Госномер</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="autoLicensePlateLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Модель авто</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Контакты</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Год выпуска</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="manufactureYearLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="VINnumberLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ФИО клиента</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="autoModelLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>VIN-номер</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="clientFMLnameLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="saveUpdatedInfo">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #dbdcf7;
	border: 1px solid #999999;
}

QPushButton:hover {
	background-color: #d5d6f0;
	border: 1px solid #acacac;
}

QPushButton:pressed {
	background-color: #cbcde6;
	border: 2px solid #999999;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
